File: week3/lab_py/bank/bank.py
class Bank:
    def main():
        inpt = input("Amount to deposit $")
        deposit = float(inpt)
        print(f'Amount ${deposit:.2f} deposited')

        inpt = input("Amount to withdraw $")
        withdraw = float(inpt)
        print(f'Amount ${withdraw:.2f} withdrawn')

if __name__ == "__main__":
    Bank.main()

File: week3/lab_java/bank/Bank.java
package lab3.bank;

import java.util.Scanner;

public class Bank {
    static Scanner in = new Scanner(System.in);

    public static void main(String[] args) {
        System.out.print("Amount to deposit $");
        float deposit = in.nextFloat();
        System.out.printf("Amount to deposit $%.2f\n",deposit);

        System.out.print("Amount to withdraw $");
        float withdraw = in.nextFloat();
        System.out.printf("Amount to withdraw is $%.2f\n", withdraw);
    }
}